<ui version="4.0" >
 <class>JavaScriptPrefsBase</class>
 <widget class="QWidget" name="JavaScriptPrefsBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>456</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="Q3ListView" name="accountList" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>1</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>175</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip" >
      <string>The list of accounts</string>
     </property>
     <property name="whatsThis" >
      <string>This is the list of available Kopete accounts. To see what scripts are active for a given account, click on that account.</string>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <property name="resizeMode" >
      <enum>Q3ListView::AllColumns</enum>
     </property>
     <column>
      <property name="text" >
       <string>For Account</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3" >
    <widget class="Q3GroupBox" name="groupBox1" >
     <property name="whatsThis" >
      <string>These options enable and disable features that impact the level of access scripts have to the Kopete API. Unless you are installing scripts from untrusted sources, it is usually OK to leave these options enabled.</string>
     </property>
     <property name="title" >
      <string>JavaScript Security Options</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="factoryEnabled" >
        <property name="text" >
         <string>Allow access to the UI &amp;factory</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="signalsEnabled" >
        <property name="text" >
         <string>Allow connecting to &amp;signals</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="treeEnabled" >
        <property name="text" >
         <string>Allow access to the QObject &amp;tree</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="writeEnabled" >
        <property name="text" >
         <string>Enable &amp;write access to QObjects</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3" colspan="2" >
    <widget class="Q3GroupBox" name="groupBox2" >
     <property name="title" >
      <string>Add / Remove</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="addScript" >
        <property name="toolTip" >
         <string>Click here to add a local file as a script</string>
        </property>
        <property name="text" >
         <string>&amp;Add Script</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="downloadScript" >
        <property name="toolTip" >
         <string>Click here to download scripts from the Internet using KNewStuff</string>
        </property>
        <property name="text" >
         <string>&amp;Download Scripts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeScript" >
        <property name="toolTip" >
         <string>Click here to uninstall the selected scripts from your system</string>
        </property>
        <property name="text" >
         <string>&amp;Uninstall Selected Scripts</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="4" >
    <widget class="QPushButton" name="editScript" >
     <property name="toolTip" >
      <string>Click here to modify properties such as the script name, author, and description</string>
     </property>
     <property name="text" >
      <string>&amp;Edit Properties</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2" >
    <widget class="QPushButton" name="configureScript" >
     <property name="toolTip" >
      <string>Click here to configure this script</string>
     </property>
     <property name="whatsThis" >
      <string>Some scripts have a configuration GUI available. This button launches that UI, allowing you to configure the script's options visually.</string>
     </property>
     <property name="text" >
      <string>&amp;Configure Script</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4" >
    <widget class="Q3ListView" name="scriptList" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>List of available scripts</string>
     </property>
     <property name="whatsThis" >
      <string>This is the list of scripts available on this system. Scripts that are enabled for the account selected are checked. You can check or uncheck a script to enable or disable it for the selected account.</string>
     </property>
     <property name="selectionMode" >
      <enum>Q3ListView::Single</enum>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <property name="resizeMode" >
      <enum>Q3ListView::NoColumn</enum>
     </property>
     <column>
      <property name="text" >
       <string>Script</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Version</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Description</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Author</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
